src/collections/_documentation/clients/java/agent.md

```diff
@@ -18,3 +18,5 @@ java -agentpath:/path/to/libsentry_agent_linux-x86_64.so -jar app.jar
 You will still need to install and configure the [Sentry Java SDK]({%- link _documentation/clients/java/index.md -%}). In addition, **you must set the** `stacktrace.app.packages` option. Only exceptions that contain at least one frame from your application will be processed by the agent. You can find details about this option [on the configuration page]({%- link _documentation/clients/java/config.md -%}#in-application-stack-frames).
 
 With the SDK configured the agent should automatically enhance your events where applicable.
+
+![Example of local variable state in the Sentry UI](/src/collections/_documentation/clients/java/agent.png)
```

Comment:
I don't think that's the right way to include the file but I also don't know what the right one is. @cameronmcefee how do we do this?